abixen-platform-core/src/test/java/com/abixen/platform/core/service/LayoutServiceTest.java

```diff
@@ -47,4 +53,44 @@ public void htmlLayoutToJson() {
         assertEquals(result, "{\"rows\":[{\"columns\":[{\"styleClass\":\"col-md-12\"}]},{\"columns\":[{\"styleClass\":\"col-md-4\"},{\"styleClass\":\"col-md-4\"},{\"styleClass\":\"col-md-4\"}]}]}");
 
     }
+    @Test
+    @Before
+    public void createLayout(){
+        String  html= "<div class=\"row\">\n" +
+                "\t<div class=\"column col-md-12\"></div>\n" +
+                "</div>\n" +
+                "<div class=\"row\">\n" +
+                "\t<div class=\"column col-md-4\"></div>\n" +
+                "\t<div class=\"column col-md-4\"></div>\n" +
+                "\t<div class=\"column col-md-4\"></div>\n" +
+                "</div>";
+        Layout sampleLayout = new Layout();
+        sampleLayout.setTitle("Test Layout");
+        sampleLayout.setIconFileName("Test Icon File Name");
+        sampleLayout.setContent(html);
+        layout =layoutService.createLayout(sampleLayout);
+        assertEquals(layout.getContent(),"{\"rows\":[{\"columns\":[{\"styleClass\":\"col-md-12\"}]},{\"columns\":[{\"styleClass\":\"col-md-4\"},{\"styleClass\":\"col-md-4\"},{\"styleClass\":\"col-md-4\"}]}]}");
+        assertEquals(layout.getIconFileName(),sampleLayout.getIconFileName());
+        assertEquals(layout.getTitle(),sampleLayout.getTitle());
+    }
+    @Test
+    public void updateLayout(){
+        Layout sampleLayout = new Layout();
+        sampleLayout.setId(layout.getId());
+        sampleLayout.setIconFileName(layout.getIconFileName());
+        sampleLayout.setTitle(layout.getTitle());
+        String html="<div class=\"row\">\n" +
+                "<div class=\"column col-md-12\"></div>\n" +
+                "</div>\n" +
+                "<div class=\"row\">\n" +
+                " <div class=\"column col-md-4\"></div>\n" +
+                "</div>";
+        layout.setContent(html);
+        Layout updatedLayout = layoutService.updateLayout(layout);
+        assertEquals(updatedLayout.getContent(),"{\"rows\":[{\"columns\":[{\"styleClass\":\"col-md-12\"}]},{\"columns\":[{\"styleClass\":\"col-md-4\"}]}]}");
```

Comment:
Should be compared like so:
assertEquals(layout.getContent(), updatedLayout.getContent()); similarly as below